### DISCUSSION TITLE
Wallet content difference with same wallet but different machines.

### DISCUSSION THREAD

**Discussion by Zillos** [Chia-Network/chia-blockchain#11520] [Support]
![Captura de pantalla 2022-05-13 155346](https://user-images.githubusercontent.com/101582623/168380330-a6939e94-4dad-4fa9-bfbc-fee817e9269a.png)
![Captura de pantalla 2022-05-13 154936](https://user-images.githubusercontent.com/101582623/168380350-2ac91895-6fd7-499f-b80e-e11f4c98f8db.png)

hello everyone, I have a problem with one of my machines. I have a machine where I have 18 Chias in the wallet, but recently I took one of the machines to another house, and in that machine now I have just 8 Chia. What's the problem here? I'm using the same wallet in both machines, is there a node problem because I'm using different routers?...

**Comment by mariano54** [comment]
What version are you using?
Do you have a local node running?
Did you wait some time after "syced" started showing?
Did you try deleting the wallet DB files? `~/.chia/wallet/db`
